tools/circleci_dependencies.sh

```diff
@@ -33,7 +33,7 @@ elif [[ "$CIRCLE_JOB" == "linkcheck"* ]]; then
 else  # standard doc build
 	echo "Installing doc build dependencies"
 	python -m pip uninstall -y pydata-sphinx-theme
-	python -m pip install --upgrade --progress-bar off --only-binary matplotlib -r requirements.txt -r requirements_testing.txt -r requirements_doc.txt
+	python -m pip install --upgrade --progress-bar off --only-binary matplotlib https://github.com/marsipu/mne-qt-browser/zipball/fix_threading -r requirements.txt -r requirements_testing.txt -r requirements_doc.txt
```

Comment:
Reminder: Needs to be reverted before merging. This PR should be merged after mne-qt-browser was bumped to the next version (0.1.7)
```suggestion
	python -m pip install --upgrade --progress-bar off --only-binary matplotlib mne-qt-browser -r requirements.txt -r requirements_testing.txt -r requirements_doc.txt
```